docs/technical-design/creating-endpoints.md

```diff
@@ -17,19 +17,36 @@ You can use the [Apollo Sandbox tool](https://www.apollographql.com/docs/graphos
 - [The Explorer](https://www.apollographql.com/docs/graphos/explorer/explorer/): A web IDE for creating, running, and managing GraphQL operations
 - Schema referencing
 
-To connect to the local graphql deployment we need to configure connection settings in the Explorer tool.
-- Visit the tool at https://studio.apollographql.com/sandbox/explorer.
-- Top left of the browser, the `SANDBOX` input field will have a gear icon.
-- Click on this gear icon to open `Connection settings` modal.
-- In the modal input our local graphql endpoint `http://localhost:3030/local/graphql` into the `Endpoint` field.
-- Under `Shared headers` we will need to configure our cognito authentication.
-   - There are two inputs in the `Shared headers`  section, the first is `header` name field and second is the `value`.
-   - For the `header` name field it should be `cognito-authentication-provider`.
-   - For the `value` field, you can paste in the following for each user's authentication.
-      - CMS User Zuko: `{"id":"user3","email":"zuko@example.com","givenName":"Zuko","familyName":"Hotman","role":"CMS_USER","stateAssignments":[]}`
-      - State User Aang: `{"id":"user1","email":"aang@example.com","givenName":"Aang","familyName":"Avatar","role":"STATE_USER","stateCode":"MN"}`
-      - State User Toph: `{"id":"user2","email":"toph@example.com","givenName":"Toph","familyName":"Beifong","role":"STATE_USER","stateCode":"VA"}`
-   - Click the save button, and you should see a green dot with our endpoint inside the `SANDBOX` input field.
+We have two ways to access the Apollo Sandbox tool:
+
+**Embedded Apollo Explorer Tool**:
+>In all environments, except `prod`, we have embedded the Apollo Explorer Tool into the MC-Review app. We embedded the tool into the app to simplify authorization configuration for `dev` and `val` environments. By doing so, we can programmatically configure the tool's authorization, eliminating the need for manual configuration by the user.
+>
+>Accessing the embedded tool:
+>- Log into the MC-Review app
+>   - Depending on the user, performing certain GraphQL operations will be restricted. The operation authorization is done in the resolvers, `services/app-api/src/resolvers`.
+>   - For example, a state user will not be allowed to unlock a submission. The GraphQL operation will return an unauthorized error.
+>- Once logged in, input this url `[hostname]/dev/graphql-explorer` to access the tool.
+
+**External Apollo Explorer Tool**:
+>Apollo Explorer is embedded in the local deployment, so there's no need to access it externally, except in rare cases like when the frontend is not accessible, and we need to perform GraphQL operations.
```

Comment:
> like when the frontend is not accessible, and we need to perform GraphQL operations.

Yea I guess this isn't a good scenario. Another one could be accessing graph operations using new user types that do have users made yet. My reason for keeping them is for those rare cases and I really don't know what they could be, but it would be good to have the option to perform those operations when they arise.

Reply:
> but it would be good to have the option to perform those operations when they arise.

This should be rephrased to: `but it would be good to have the instructions on how to perform those operations outside the frontend app when they arise.`